src/doc/book/documentation.md

```diff
@@ -9,7 +9,7 @@ document your project.
 The Rust distribution includes a tool, `rustdoc`, that generates documentation.
 `rustdoc` is also used by Cargo through `cargo doc`.
 
-Documentation can be generated in two ways: from source code, and from
+Documentation can be generated in two ways; from source code, and from
```

Comment:
Yes, Me too. First of all, this is an independent clause, and is very commonly used like this, with a list.